test/metabase/driver/sql_jdbc/sync/describe_database_test.clj

```diff
@@ -221,7 +222,10 @@
                    (mt/db)
                    nil
                    (fn [^java.sql.Connection conn]
-                     (.setAutoCommit conn auto-commit)
+                     (try
+                       (.setAutoCommit conn auto-commit)
+                       (catch Exception _
+                         (log/trace "Failed to set auto commit.")))
```

Comment:
Why are we throwing out this exception?

Reply:
I've added a comment. Databricks does not support setting `setAutoCommit` to `false`. This way we test the `true` value only.